zyte_spider_templates_project/settings.py

```diff
@@ -6,28 +6,28 @@
 BOT_NAME = "zyte_spider_templates_project"
 
 SPIDER_MODULES = [
-    "zyte_spider_templates.spiders",
     "zyte_spider_templates_project.spiders",
 ]
 NEWSPIDER_MODULE = "zyte_spider_templates_project.spiders"
 
-CLOSESPIDER_TIMEOUT_NO_ITEM = 900
 ADDONS = {
+    "zyte_spider_templates.Addon": 100,
     "scrapy_zyte_api.Addon": 500,
     "duplicate_url_discarder.Addon": 600,
 }
-SCHEDULER_DISK_QUEUE = "scrapy.squeues.PickleFifoDiskQueue"
-SCHEDULER_MEMORY_QUEUE = "scrapy.squeues.FifoMemoryQueue"
+
+DOWNLOADER_MIDDLEWARES = {
+    "scrapy_poet.InjectionMiddleware": 543,
+    "scrapy_zyte_api.ScrapyZyteAPIDownloaderMiddleware": 1000,
```

Comment:
Do we scrapy_zyte_api iddlewares here, as we have scrapy_zyte_api.Addon enabled?

Reply:
`zyte_spider_templates.Addon` does not set `ScrapyZyteAPIDownloaderMiddleware`, but `scrapy_zyte_api.Addon` does, so we don’t need it here. Same for `scrapy_zyte_api.ScrapyZyteAPISpiderMiddleware` below.

Reply:
Fixed!